requirements.in

```diff
@@ -1,35 +1,35 @@
 # The file contains the direct ckan requirements.
 # Use pip-compile to create a requirements.txt file from this
 Babel==2.3.4
-Jinja2==2.8
-Pylons==0.9.7
 Beaker==1.7.0 # need to pin this because of https://github.com/bbangert/beaker/commit/fc511ceda4305fcf54919b3f081fed7790e2fef3
-bleach==1.4.2
-WebTest==1.4.3  # need to pin this so that Pylons does not install a newer version that conflicts with WebOb==1.0.8
+bleach==1.4.3
 fanstatic==0.12
-Markdown==2.4
-ofs==0.4.1
+Flask==0.10.1
+Jinja2==2.8
+Markdown==2.6.6
+ofs==0.4.2
 ordereddict==1.1
 Pairtree==0.7.1-T
+passlib==1.6.5
 paste==1.7.5.1
-passlib==1.6.2
 psycopg2==2.4.5
-pysolr==3.4.0
+pysolr==3.5.0
+Pylons==0.9.7
 python-dateutil>=1.5.0,<2.0.0
-pyutilib.component.core==4.5.3
+pytz==2016.4
+pyutilib.component.core==4.6.4
 repoze.who-friendlyform==1.0.8
-repoze.who==2.0
-requests==2.7.0
+repoze.who==2.3
+requests==2.10.0
 Routes==1.13
-sqlalchemy-migrate==0.9.1
+sqlalchemy-migrate==0.10.0
 SQLAlchemy==0.9.6
```

Comment:
Did you left out SQLAlchemy for reason in particular? It's a critical one so it can probably go into a separate PR